.travis.yml

```diff
@@ -26,6 +26,14 @@ matrix:
       env:
         - MATRIX_EVAL="CXX=c++"
 
+#=============================================================================
+# Cache directories
```

Comment:
nice! I saw this in my periphery, but didn't get a chance to look into it. Very useful.